app/grandchallenge/notifications/views.py

```diff
@@ -48,6 +54,22 @@ def update(self, request, *args, **kwargs):
         return response
 
 
+class FollowViewSet(
+    mixins.DestroyModelMixin, viewsets.ReadOnlyModelViewSet,
+):
+    queryset = Follow.objects.all()
+    serializer_class = FollowSerializer
+    permission_classes = (DjangoObjectPermissions,)
```

Comment:
Are there tests for the permissions and functionality?

Reply:
There are now! Sorry, I was under the impression that I had added them, but I apparently didn't.